packages/wdio-config/src/lib/ConfigParser.ts

```diff
@@ -73,10 +71,8 @@ export default class ConfigParser {
         /**
          * on launcher compile files if Babel or TypeScript are installed using our defaults
          */
-        if (!loadAutoCompilers(this._autocompileConfig, this._moduleRequireService)) {
-            if ( this._autocompileConfig.autoCompile ) {
-                log.debug('No compiler found, continue without compiling files')
-            }
+        if (this._config.autoCompileOpts && !loadAutoCompilers(this._config.autoCompileOpts!, this._moduleRequireService)) {
```

Comment:
this is just for my education: what does this condition mean?  does it mean if the autoCompile options are set but the auto compilers are not loaded?

Reply:
yeah .. `this._config.autoCompileOpts` is optional even though it will be always filled with a default value. Types could be improved here but in order to make the compiler happy I made an additional check here.